light-client/src/main/java/tech/pegasys/teku/lightclient/client/SyncCommittee.java

```diff
@@ -0,0 +1,17 @@
+package tech.pegasys.teku.lightclient.client;
+
+import tech.pegasys.teku.bls.impl.PublicKey;
+
+public class SyncCommittee {
+	
+	  private PublicKey pubKey;
```

Comment:
Typically in Teku we prefer immutable data structures since it avoids a lot of thread safety issues.  It's hard to say whether this should be immutable or not given there isn't anything really using it at the moment though.

Reply:
This is indeed meant to be immutable. The basic premise is to get and set said PublicKeys (was in the process of building out the API). I'm using Lodestar's light-client build as reference btw (https://github.com/ChainSafe/lodestar/tree/master/packages/light-client)